### DISCUSSION TITLE
moveTo evolution: dodging actors with collider set

### DISCUSSION THREAD

**Discussion by cirossmonteiro** [excaliburjs/Excalibur#2617] [Ideas]
Does it make sense?

**Comment by esonarhe** [comment]
@cirossmonteiro Are you thinking that if a collider is set on one actor acting as a wall, and another acting as a player/npc it would path around the wall?

**Comment by cirossmonteiro** [comment]
Exactly

Em qua., 5 de abr. de 2023 13:14, Erik Onarheim ***@***.***>
escreveu:
